builder/azure/arm/authenticate_secret_test.go

```diff
@@ -22,13 +21,13 @@ func TestNewSecretOAuthTokenProvider(t *testing.T) {
 	if spn.Token().RefreshToken != "" {
 		t.Errorf("spn.Token().RefreshToken: expected=\"\", actual=%s", spn.Token().RefreshToken)
 	}
-	if spn.Token().ExpiresIn != "" {
+	if spn.Token().ExpiresIn != "0" {
 		t.Errorf("spn.Token().ExpiresIn: expected=\"\", actual=%s", spn.Token().ExpiresIn)
```

Comment:
If the expected value changed, please also update the error message in case the test fails.

Reply:
x3